src/dxss/solve_1d.py

```diff
@@ -50,10 +50,10 @@ def __init__(self, Asp, psolver):  # noqa: N803 | convention Ax = b
             )
 
     def solve(self, b_inp, x_out):
-        self.solver._check_A(self.Asp)
-        b = self.solver._check_b(self.Asp, b_inp.array)
+        self.solver.check_A(self.Asp)
```

Comment:
```suggestion
        self.solver._check_A(self.Asp)  # noqa: SLF001, TODO: fix this.
```
You can't just rename the method!?